rpm-package/jellyfin.spec

```diff
@@ -0,0 +1,152 @@
+%global         debug_package %{nil}
+# jellyfin tag to package
+%global         gittag v10.0.0
+# Taglib-sharp commit of the submodule since github archive doesn't include submodules
+%global         taglib_commit ee5ab21742b71fd1b87ee24895582327e9e04776
+%global         taglib_shortcommit %(c=%{taglib_commit}; echo ${c:0:7})
+
+Name:           jellyfin
+Version:        10.0.0
+Release:        1%{?dist}
+Summary:        The Free Software Media Browser.
+License:        GPLv2
+URL:            https://jellyfin.media
+Source0:        https://github.com/%{name}/%{name}/archive/%{gittag}.tar.gz
+Source1:        jellyfin.service
+Source2:        jellyfin.env
+Source3:        jellyfin.sudoers
+Source4:        restart.sh
+Source5:        https://github.com/mono/taglib-sharp/archive/%{taglib_commit}/taglib-sharp-%{taglib_shortcommit}.tar.gz
+Source6:        update-db.sh
+Source7:        jellyfin-firewalld.xml
+
+%{?systemd_requires}
+BuildRequires:  systemd
+Requires(pre):  shadow-utils
+BuildRequires:  libcurl-devel, fontconfig-devel, freetype-devel, openssl-devel, glibc-devel, libicu-devel
+Requires:       libcurl, fontconfig, freetype, openssl, glibc libicu
+# Requirements not packaged in main repos
+# COPR @dotnet-sig/dotnet
+BuildRequires:  dotnet-sdk-2.2
+# RPMfusion free
+Requires:       ffmpeg
+
+# For the update-db-paths.sh script to fix emby paths to jellyfin
+%{?fedora:Recommends: sqlite}
+
+# Fedora has openssl1.1 which is incompatible with dotnet 
+%{?fedora:Requires: compat-openssl10}
+# Disable Automatic Dependency Processing for Centos
+%{?el7:AutoReqProv: no}
+
+%description
+Jellyfin is a free software media system that puts you in control of managing and streaming your media.
+
+
+%prep
+%autosetup -n %{name}-%{version}
+pushd ThirdParty
+    tar xf %{S:5}
+    rm -rf taglib-sharp
+    mv taglib-sharp-%{taglib_commit} taglib-sharp
+popd
+
+%build
+export DOTNET_CLI_TELEMETRY_OPTOUT=1
+export DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1
+dotnet build --runtime linux-x64
+
+%install
+export DOTNET_CLI_TELEMETRY_OPTOUT=1
+export DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1
+dotnet publish --configuration Release --output='%{buildroot}%{_libdir}/jellyfin' --self-contained --runtime linux-x64
+%{__install} -D -m 0644 LICENSE %{buildroot}%{_datadir}/licenses/%{name}/LICENSE
+%{__install} -D -m 0644 debian/conf/jellyfin.service.conf %{buildroot}%{_sysconfdir}/systemd/system/%{name}.service.d/override.conf
```

Comment:
this doesn't feel right when `RPM` package references stuff from `debian`... if they're the same they should live in some common subdir IMHO

Reply:
Agreed, or just two copies - this is one of those things that should never change barring major rewrites of the packaging.

Reply:
I removed all the references to the debian package and directly referenced the source for `logging.json` as for the `override.conf` i copied the one from the debian package as it only serves as a sort of placeholder for the user to override the user under wich jellyfin runs.